src/Pact/Repl/Lib.hs

```diff
@@ -111,12 +111,12 @@ replDefs = ("Repl",
       "Set transaction JSON data, either as encoded string, or as pact types coerced to JSON."
 
      ,defZRNative "continue-pact" continuePact
-      (funType tTyString [("pact-id",tTyInteger),("step",tTyInteger)] <>
-       funType tTyString [("pact-id",tTyInteger),("step",tTyInteger),("rollback",tTyBool)] <>
-       funType tTyString [("pact-id",tTyInteger),("step",tTyInteger),("rollback",tTyBool),("yielded",tTyObject (mkSchemaVar "y"))])
-      [LitExample "(continue-pact 2 1)", LitExample "(continue-pact 2 1 true)",
-       LitExample "(continue-pact 2 1 false { \"rate\": 0.9 })"]
-      ("Continue previously-initiated pact identified by PACT-ID at STEP, " <>
+      (funType tTyString [("tx-id",tTyInteger),("chain-id",tTyString),("step",tTyInteger)] <>
+       funType tTyString [("tx-id",tTyInteger),("chain-id",tTyString),("step",tTyInteger),("rollback",tTyBool)] <>
+       funType tTyString [("tx-id",tTyInteger),("chain-id",tTyString),("step",tTyInteger),("rollback",tTyBool),("yielded",tTyObject (mkSchemaVar "y"))])
+      [LitExample "(continue-pact 2 \"\" 1)", LitExample "(continue-pact 2 1 true)",
```

Comment:
Not sure if this is the best user experience

Reply:
well, we're trying to minimize the number of `Lib` functions so overloading is one way to do that, but open to suggestions

Reply:
Aw gotcha. Originally I was wondering if I should lean into overloading and default to "" for chain-id. But I changed this function a bit to just take in the pact-id, so I'm going to resolve this.